lms/djangoapps/instructor/views/registration_codes.py

```diff
@@ -0,0 +1,117 @@
+"""
+E-commerce Tab Instructor Dashboard Query Registration Code Status.
+"""
+from django.core.urlresolvers import reverse
+from django.views.decorators.http import require_POST
+from instructor.enrollment import get_email_params, send_mail_to_student
+from django.utils.translation import ugettext as _
+from courseware.courses import get_course_by_id
+from instructor.views.api import require_level
+from student.models import CourseEnrollment
+from util.json_request import JsonResponse
+from shoppingcart.models import CourseRegistrationCode, RegistrationCodeRedemption
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from django.views.decorators.cache import cache_control
+import logging
+
+log = logging.getLogger(__name__)
+
+
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+@require_POST
```

Comment:
I don't why this block has been closed. But you can review this on the file changes tab. I have made that change.